Kernel/Devices/PCISerialDevice.cpp

```diff
@@ -5,6 +5,8 @@
  */
 
 #include <Kernel/Devices/PCISerialDevice.h>
+#include <Kernel/UnixTypes.h>
+#include <LibC/sys/ttydefaults.h>
```

Comment:
Including a userland header (LibC) in kernel code is generally a bad idea

Reply:
We are already doing this for other headers too in `TTY.cpp`. Would it be good if that refactoring came as a separate PR?

Reply:
sure